src/api/edge/imagesInfo.js

```diff
@@ -0,0 +1,120 @@
+export const EDGE_API = '/api/edge/v1';
+import { instance } from '@redhat-cloud-services/frontend-components-utilities/interceptors/interceptors';
+import { useAxiosWithPlatformInterceptors } from '@redhat-cloud-services/frontend-components-utilities/interceptors';
+export const axiosInstance = () => useAxiosWithPlatformInterceptors();
+export const getImageSet = ({
+  id,
+  params = {
+    limit: 10,
+    offset: 0,
+    sort_by: '-created_at',
+  },
+}) => {
+  const query = getTableParams(params);
+  return axiosInstance.get(`${EDGE_API}/image-sets/${id}?${query}`);
+};
```

Comment:
This should rather be something like:

```js
 const useGetImageSet = () => {
   const axios = useAxiosWithPlatformInterceptors();
   
   return (id) => 
    axios.get(`${EDGE_API}/image-sets/view/${id}`);
 }
```

to be a proper hook and not break React's rules of hooks.

In components where `getImageSet` needs to be called the function returned by the `useGetImageSet` hook should be called, like:


```js
const Components = () => {
    const getImageSet = useGetImageSet();

    ....
    useEffect(() => {
      (async () => {
        const resp = await getImageSet(params);
        ...
      })();
    }, []);

   return <>...</>
}

Reply:
This ^ applies to all functions in this file.